doc/changelog/04-tactics/14137-setoid-rewrite-type.rst

```diff
@@ -0,0 +1,7 @@
+- **Added:**
+  support for monomorphic and polymorphic generalized rewriting with
+  relations in Type for :tacn:`setoid_rewrite`.
```

Comment:
Perhaps invert this so the concrete detail (it's for setoid_rewrite) appears first?  I think a little easier to read.

```suggestion
  :tacn:`setoid_rewrite` new supports monomorphic and polymorphic generalized
  rewriting with relations in Type.
```

Reply:
How about this?
The "support for" follows from the "Added" section. The feature is "generalized rewriting" and the tactic to do this is "rewrite" or "setoid_rewrite" (the first calls the second).

```suggestion
support for generalized rewriting (via the :tacn:`setoid_rewrite` tactic) for
monomorphic and polymorphic relations in Type.
```

Reply:
Thanks for taking a look!

Reply:
I'd leave out the boilerplate phrase "support for".  I consider the "Added" as a title rather than the beginning of a sentence (so not read as "Added support for generalized rewriting ...").

Maybe "generalized rewriting for relations in Type using :tacn:\`setoid_rewrite\` and :tacn:\`rewrite`"?  Wouldn't "monomorphic and polymorphic" be understood or expected?

It doesn't have to perfect.